frontend/quasar.conf.js

```diff
@@ -43,11 +43,17 @@ module.exports = function (ctx) {
       //            (not treeshaking Quasar; biggest bundle size; convenient)
       all: 'auto',
 
-      components: [],
-      directives: [],
+      components: [
+
+      ],
+      directives: [
+        
+      ],
 
       // Quasar plugins
-      plugins: []
+      plugins: [
+        'Dialog',
```

Comment:
I believe `dialog` doesn't need to be here as it's part of quasar and not a quasar plugin!

Reply:
It's needed since I'm using a custom component (ClientPopup.vue) instead of the default component for the dialog box.